lab-carlo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-carlo",
+  "version": "1.0.0",
+  "description": "class 7 lab",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha chai",
```

Comment:
This should be just `"test": "mocha"`. To use mocha you call `mocha <files>`, if you don't give it any files it will assume you want the `test/` directory.